debs/xenial/install.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+# set -o xtrace
+
+readonly ROOT="$(cd "$(dirname "${BASH_SOURCE}")/../../" && pwd)"
+
```

Comment:
LOCAL_REPOSITORY should to be defined as `false` if not defined previously:

<pre>
LOCAL_REPOSITORY=${LOCAL_REPOSITORY:-false}
</pre>